opportunity/README.md

```diff
@@ -133,8 +137,20 @@ To conform to the Create Order spec, use `"method": "POST"`.
 
 If no Body parameters apply to an Opportunity, use `"body": {}`.
 
-An Opportunity's links **must** include a `rel=create-order` link that allows
-the user to Order this opportunity.
+It is **strongly recommended** to include include a `rel=create-order` link on
+an Opportunity to allows the user to order the Opportunity. Consider the
+inclusion of this link a **must** where ordering of an individual Opportunity
+is supported by the given Product. Omission of this link is valid when
+Opportunities are considered informational-only and cannot be ordered directly
+(i.e., the precision of the Opportunities exceeds the implementation's limits
+with regard to valid order parameters, such as when orders have a minimum
+datetime range greater than any individual Opportunity's datetime range).
+
+When Opportunity `create-order` links are not included then it is **required**
+to include the `create-order` link on the OpportunityCollection. Similarly,
+when a `create-order` link is not included at the OpportunityCollection level
```

Comment:
This sentence seems to just duplicate the info above staring with "Consider the inclusion..."